main/appengine_config.py

```diff
@@ -6,6 +6,10 @@
 if os.environ.get('SERVER_SOFTWARE', '').startswith('Google App Engine'):
   sys.path.insert(0, 'lib.zip')
 else:
+  if os.name == 'nt':
+    os.name = None
+    sys.platform = ''
```

Comment:
@dhstack Will you do that..? and post here the issue numbers so we could vote/subscribe..